external/fv3fit/fv3fit/emulation/layers/fields.py

```diff
@@ -42,31 +45,65 @@ def call(self, tensor: tf.Tensor):
 
         return tensor
 
+    def get_config(self):
+
+        config = super().get_config()
+
+        if self._selection is not None:
+            serializable_selection = [
+                self._selection.start,
+                self._selection.stop,
+                self._selection.step,
+            ]
+        else:
+            serializable_selection = None
+
+        config.update(
+            {"norm_layer": self.normalize, "selection": serializable_selection}
```

Comment:
Normalize layers are already serializable so passing into the config dictionary prevents from having to re-fit the layers with a sample tensor.